packages/discord.js/typings/index.d.ts

```diff
@@ -6319,7 +6324,7 @@ export interface MessageCreateOptions extends BaseMessageOptionsWithPoll {
   tts?: boolean;
   nonce?: string | number;
   enforceNonce?: boolean;
-  reply?: ReplyOptions;
+  messageReference?: MessageReference & { failIfNotExists?: boolean };
```

Comment:
Doesn't seem like `failIfNotExists` is documented in the JSDoc for being allowed in `messageReference`